mopidy_listenbrainz/frontend.py

```diff
@@ -15,14 +19,142 @@ class ListenbrainzFrontend(pykka.ThreadingActor, CoreListener):
+
+        existing_playlists = self.playlists.as_list().get()
+        recommendation_playlist_uri_prefix = (
+            "listenbrainz:playlist:recommendation"
+        )
+        filtered_existing_playlists = dict(
+            [
+                (ref.uri, ref)
+                for ref in existing_playlists
+                if ref.uri.startswith(recommendation_playlist_uri_prefix)
+            ]
+        )
+
+        for playlist_data in playlist_datas:
+            source = playlist_data.playlist_id
+            playlist_uri = f"{recommendation_playlist_uri_prefix}:{playlist_data.playlist_id}"
+            tracks = self._collect_playlist_tracks(playlist_data)
+
+            if len(tracks) == 0:
+                logger.debug(
+                    f"Skipping import of playlist with no known track for {source!r}"
+                )
+                continue
+
+            if playlist_uri in filtered_existing_playlists:
+                filtered_existing_playlists.pop(playlist_uri)
+                # must pop since filtered_existing_playlists will
+                # finally be deleted
+
+                logger.debug(f"Already known playlist {playlist_uri}")
+                # maybe there're new tracks in Mopidy's database...
+            else:
+                query = self.playlists.create(
+                    name=playlist_uri, uri_scheme="listenbrainz"
+                )
+                # Hack, hack: The backend uses first parameter as URI,
+                # not name...
+                playlist = query.get()
+                if playlist is None:
+                    logger.warning(f"Failed to create playlist for {source!r}")
+                    continue
+
+                logger.debug(f"Playlist {playlist.uri!r} created")
+
+            complete_playlist = Playlist(
+                uri=playlist_uri,
+                name=playlist_data.name,
+                tracks=tracks,
+                last_modified=playlist_data.last_modified,
+            )
+            query = self.playlists.save(complete_playlist)
+            playlist = query.get()
+            if playlist is None:
+                logger.warning(f"Failed to save playlist for {source!r}")
+            else:
+                import_count += 1
+                logger.debug(
+                    f"Playlist saved with {len(playlist.tracks)} tracks {playlist.uri!r}"
+                )
+
+        for playlist in filtered_existing_playlists.values():
+            logger.debug(f"Deletion of obsolete playlist {playlist.uri!r}")
+            self.playlists.delete(playlist.uri)
+
+        logger.info(
+            f"Successfully imported ListenBrainz playlists: {import_count}"
+        )
+        self._schedule_playlists_import()
+
+    def _collect_playlist_tracks(
+        self, playlist_data: PlaylistData
+    ) -> List[Track]:
+        tracks: List[Track] = []
+        search_schemes = self.config["listenbrainz"].get("search_schemes", [])
```

Comment:
```suggestion
        search_schemes = self.config["listenbrainz"].get("search_schemes", ["local:"])
```
keep this consistent with default settings?

alternately, according to the docs, a value of `None` (distinct from `[]`?) indicates "use all URLs," if we want that to be the default.

```suggestion
        search_schemes = self.config["listenbrainz"].get("search_schemes", None)
```